src/python/ddapp/roboturdf.py

```diff
@@ -185,7 +186,7 @@ def removeFromView(self, view):
         view.render()
 
 
-def loadRobotModel(name, view=None, parent='planning', urdfFile=None, color=None, visible=True):
+def loadRobotModel(name, view=None, parent='planning', urdfFile=None, color=None, visible=True, colorMode='URDF Colors'):
```

Comment:
I don't think colorMode should default to URDF Colors until the option is switchable by the user.